app/templates/events/view.hbs

```diff
@@ -35,22 +35,26 @@
       </div>
     </div>
   </div>
-
   <div class="row" style="padding-top: 15px">
     <div class="sixteen wide column">
-      <div class="ui fluid pointing stackable menu {{unless device.isMobile 'secondary'}}">
-        {{#link-to 'events.view.index' class='item'}}
+      {{#events/dropdown-navigation}}
```

Comment:
Since this not just for events page and it is primarily a tabbed navigation, we can call this component `tabbed-navigation`